news/2023/2023-09-25-osulazer-updates-september-24.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "osu!(lazer) Updates: September 24"
+date: 2023-09-25 21:00:00 +0000
+---
+
+Marvel at the never-ending stream of usability improvements.
+
+![](/wiki/shared/news/2023-09-25-osulazer-updates-september-24/banner.jpg)
+
+Things are progressing faster towards the goal of leaderboards by the end of the year, but so is the to-do list of getting there. Find out all about it in this video here or read the executive summary below:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/SWEE8XlGluI" frameborder="0" allowfullscreen></iframe>
+</div>
+
+### Changes
+
+- **Velocity change visualisation toggle in the editor.** In osu!mania and osu!taiko, the speed of objects can be toggled to be constant or reflect gameplay.
+- **-10 approach rate for the Difficulty Adjust mod.** Courtesy of reading expert [-GN](https://osu.ppy.sh/users/895581), you may now attempt and fail something he does with ease.
```

Comment:
at a glance this sounds like u can just set it to -10, not anything in between 0 and -10